app/DoctrineMigrations/Version20200416135127.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20200416135127 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE VIEW `view_ra_candidate` AS
+(
+SELECT DISTINCT i.id          as identity_id,
+                i.institution,
+                i.common_name,
+                i.email,
+                i.name_id,
+                a.institution AS ra_institution
+FROM identity i
+         INNER JOIN vetted_second_factor vsf on vsf.identity_id = i.id
+         INNER JOIN institution_authorization a ON (
+        a.institution_role = \'select_raa\' AND a.institution_relation = i.institution
+    )
+WHERE NOT EXISTS(SELECT 1 FROM ra_listing AS l WHERE l.identity_id = i.id AND l.ra_institution = a.institution)
+);');
```

Comment:
3 nitpicks:
1. For the sake of readability consider using HEREDOC to capture the query.
2. My inner SQL linter triggered on mixed-case use of MySQL keywords (`AS` and `as`, `on`). You also not use them consistently. For the aliases you create for the table names (sh|c)ould also have an `AS` keyword.
3. Consider changing the where not exists subquery into a multi-liner.

Reply:
Done